CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Bug Fixes
 
 ### Breaking Changes
+* (sdk) [\#187](https://github.com/finschia/finschia/pull/187) Bump github.com/Finschia/finschia-sdk from v0.47.0 to v0.47.1-0.20230517010045-e9fe90608161
```

Comment:
How about changing title and chagnelog information to detail function information, because this bump up apply just one commit change?

Reply:
OK, I will update it.